gulpfile.js

```diff
@@ -0,0 +1,259 @@
+const fs = require("fs");
```

Comment:
It's already in README.

Reply:
No, but still, we need to comment everything in case it does need to be touched. Even if it's in the README, we need to comment the file itself so everyone knows exactly what each line of code does

Reply:
Should I JSDoc it or comment line-by-line?

Reply:
Line-by-line, there are no exported functions here.